### PR TITLE
Fix/calculate curl

### PR DESCRIPTION
**Pull request by KlausBSautter**
as discussed today

### CODE REVIEW

File: GUI/top_gui_functions.py [hunk 1]
```diff
@@ -985,7 +1007,7 @@ def _PlotMesh(self,plot_a):
                     [ (self.systemData.Nodes[nodeId1,1], self.systemData.Nodes[nodeId1,2]) ,
                       (self.systemData.Nodes[nodeId2,1], self.systemData.Nodes[nodeId2,2]) ]
                 )
-
+        self._Check_Curl()
```
Comment (armingeiser) [inline GUI/top_gui_functions.py]: i think this check could already be done at the ActionOK callback of the window that creates the geometry.
There we would have to check only the for nodes entered by the user, and either reorder them or show an error message.

Comment (armingeiser) [reply GUI/top_gui_functions.py]: The check for each element happens then anyway with the new functions in the FE solution

**Comment by KlausBSautter** [comment]
@armingeiser I guess this does the job

File: GUI/top_gui_functions.py [hunk 2]
```diff
@@ -1954,11 +1935,25 @@ def _Action_OK(self,x1,x2,x3,x4,y1,y2,y3,y4,elements_a,elements_b):
             self.master._ComputeMeshRectangle(edge_node_coordinates,edge_element_count)
             self.master._UpdateVisualization()
 
-        else:
+        elif (curl_flag == True):
             if (elements_a.get() == 0): self.E1['fg'] = 'red'
             else: self.E1['fg'] = 'black'
             if (elements_b.get() == 0): self.E2['fg'] = 'red'
             else: self.E2['fg'] = 'black'
+        else: pass
```
Comment (armingeiser) [inline GUI/top_gui_functions.py]: I think the else statement should still set the colors of element_a or element_b

**Pull request merged**